pages/precompiled.tsx

```diff
@@ -1,41 +1,31 @@
-import fs from 'fs'
-import path from 'path'
+import { useContext } from 'react'
 
-import matter from 'gray-matter'
 import type { NextPage } from 'next'
-import { serialize } from 'next-mdx-remote/serialize'
-import getConfig from 'next/config'
-import Link from 'next/link'
-import { IPrecompiledDocs, IPrecompiledGasDocs } from 'types'
+import { IOpcodeDocs, IOpcodeGasDocs } from 'types'
 
-import { GITHUB_REPO_URL } from 'util/constants'
+import { EthereumContext } from 'context/ethereumContext'
 
+import ContributeBox from 'components/ContributeBox'
 import HomeLayout from 'components/layouts/Home'
-import PrecompiledReferenceTable from 'components/PrecompiledReference'
-import { H1, H2, Container, Button } from 'components/ui'
+import ReferenceTable from 'components/Reference'
+import { H1, H2, Container, RelativeLink as Link } from 'components/ui'
 
-const docsDir = 'docs/precompiled'
-
-const { serverRuntimeConfig } = getConfig()
-
-const ILink = ({ link, title }: { link?: string; title: string }) => (
-  <Link href={link ? `/${link}` : '/'} passHref>
-    <a className="underline font-mono">{title}</a>
-  </Link>
-)
+import generateDocs from './static/generateDocs'
 
 // It seems the memory expansion computation and constants did not change since frontier, but we have to keep an eye on new fork to keep this up to date
 const PrecompiledPage = ({
-  precompiledDocs,
+  opcodeDocs,
```

Comment:
Calling it opcode kinda triggers me. Shall we just call it "docs"? And same for other places, not use opcodes when it may also be a precompiled